WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -285,19 +316,19 @@ open class QuickStartTourGuide: NSObject {
     static func customizeListTours(for blog: Blog) -> [QuickStartTour] {
         return [
             QuickStartCreateTour(),
-            QuickStartSiteTitleTour(blog: blog),
-            QuickStartSiteIconTour(),
-            QuickStartEditHomepageTour(),
+//            QuickStartSiteTitleTour(blog: blog),
+//            QuickStartSiteIconTour(),
+//            QuickStartEditHomepageTour(),
```

Comment:
Don't forget to revert these and the grow your audience ones 👍

Reply:
🙈 Fixed! b1e49c7